src/server/models/listItem.ts

```diff
@@ -51,74 +44,54 @@ async function createCountry(country: string): Promise<Country> {
 async function getPlaceGeoPoint(props: {
   countryName?: string;
   text?: string;
-}): Promise<Point | undefined> {
+}): Promise<Point> {
   const { countryName, text } = props;
 
   if (text === undefined || countryName === undefined) {
-    return undefined;
+    return [0.0, 0.0];
   }
 
   try {
-    const place = await geoLocatePlaceByText(`${text}, ${countryName}`);
-    return place?.Geometry?.Point ?? undefined;
+    const { Geometry } = await geoLocatePlaceByText(`${text}, ${countryName}`);
+
+    if (Geometry.Point !== undefined) {
+      return Geometry.Point;
+    }
+
+    return [0.0, 0.0];
   } catch (error) {
-    return undefined;
+    return [0.0, 0.0];
```

Comment:
Log added.

The only error thrown will be from the AWS service if it can't connect - if it can't find anything it'll just return an empty response rather than throw.